debian/confluent-kafka-rest.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Streaming SQL engine for Apache Kafka
+Documentation=http://docs.confluent.io/
+After=network.target confluent-kafka.target
```

Comment:
add confluent-schema-registry.target as well and any other deps you have
This is not forcing them to start but defines the startup order if those services are enabled locally.